WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Package Details/Package Selection/ShippingLabelPackageList.swift

```diff
@@ -0,0 +1,73 @@
+import SwiftUI
+import Yosemite
+
+struct ShippingLabelPackageList: View {
+    @EnvironmentObject private var viewModel: ShippingLabelPackageDetailsViewModel
+    @Environment(\.presentationMode) var presentation
+
+    var body: some View {
+        ScrollView {
+            VStack(spacing: 0) {
```

Comment:
What's the reason for this convo(rather than List)? Specific customization or something different? Maybe a comment would be useful 😇

Reply:
Yes, as you said it's just for customization purposes. `List` doesn't allow us to change easily some UI things, like the separators. c568aabb3a7f923dcdb7ee70ff4eaac4e962ea17